core/broker/broker.go

```diff
@@ -135,6 +136,10 @@ func (b *broker) Init(c *component.Component) error {
 	b.Component.SetStatus(component.StatusHealthy)
 	if b.Component.Monitor != nil {
 		b.monitorStream = b.Component.Monitor.NewBrokerStreams(b.Identity.Id, b.AccessToken)
+
+		if len(viper.GetStringMapString("monitor-servers")) > 0 {
```

Comment:
What do you suggest then to avoid running this goroutine (with an infinite loop inside) when there is no monitor-servers specified?

Here there is a similar case reading from config elsewhere that's not cmd: [core/component/status.go](https://github.com/TheThingsNetwork/ttn/blob/4db1a6ff72d07f03f5e194d28ab44439bdf1648a/core/component/status.go#L90)